build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
+def isTravis = System.getenv("TRAVIS") != null
+def isAppVeyor = System.getenv("APPVEYOR") != null
+def isCi = isTravis || isAppVeyor
+
 task createConfigs {
     description "Sets up the project by obtaining necessary files and configurations points."
     group "Setup"
+    def testTemplateFilename = isCi ? "test.ci.properties" : "test.template.properties"
```

Comment:
I was following along the conversation on the issue and based it on https://github.com/TEAMMATES/teammates/issues/6812#issuecomment-286364836, i.e., Travis should use a different unique ID than the default so that hardcoding can be detected. Also, we can tweak this file to fine-tune performance within the CI environment (e.g. optimal timeout value), independent of the default values we'd like all developers to start from.